lp-234 : add term to LimitedPartnership

lduranteau opened the pull request:
JIRA link
https://companieshouse.atlassian.net/browse/LP-234

Change description
add term to LimitedPartnership

markpit commented:
Can Term be added to the test? - https://github.com/companieshouse/api-sdk-node/blob/master/test/services/limited-partnerships/limited.partnerships.spec.ts#L44

lduranteau commented:
> Can Term be added to the test? - https://github.com/companieshouse/api-sdk-node/blob/master/test/services/limited-partnerships/limited.partnerships.spec.ts#L44

Updated and moved data to send to the correct place
